tests/driver_lis2dh12/main.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the LIS2DH12 accelerometer driver
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "fmt.h"
+#include "xtimer.h"
+#include "lis2dh12.h"
+#include "lis2dh12_params.h"
+
+
+#define DELAY       (100 * US_PER_MS)
+
+/* allocate some memory for holding the formated sensor output */
+static char str_out[3][8];
+
+/* allocate device descriptor */
+static lis2dh12_t dev;
+
+int main(void)
+{
+    xtimer_ticks32_t last_wakeup = xtimer_now();
```

Comment:
I propose moving this in front of the while queue so the following puts() are not part of the first interval...